src/Middleware/HttpsPolicy/src/HttpsRedirectionMiddleware.cs

```diff
@@ -35,13 +35,13 @@ public class HttpsRedirectionMiddleware
     /// <param name="loggerFactory"></param>
     public HttpsRedirectionMiddleware(RequestDelegate next, IOptions<HttpsRedirectionOptions> options, IConfiguration config, ILoggerFactory loggerFactory)
     {
-        _next = next ?? throw new ArgumentNullException(nameof(next));
-        _config = config ?? throw new ArgumentNullException(nameof(config));
+        ArgumentNullException.ThrowIfNull(next);
+        ArgumentNullException.ThrowIfNull(options);
```

Comment:
super nit nit; this reverses the exception if `config` and `options` are both null - previously we would have reported `"config"`, not `"options"`

I... I think I might have invented a new sub-basement of "petty nit"